lms/product/moodle/_plugin/misc.py

```diff
@@ -26,6 +26,20 @@ def get_assignment_configuration(
             # Get the config from the DL
             return self._assignment_config_from_deep_linked_config(deep_linked_config)
 
+        if (
```

Comment:
Like when we first wrote this could be refactored do less comparisons and save a few lines of code.

I reckon this very verbose style is better here.